### PR TITLE
Github style following followers

### PR DESCRIPTION
**Pull request by istiak101**
Removed tabbed following and followers from user page and added Github style following followers.

### CODE REVIEW

**Comment by Gusted** [comment]
image
The CSS is currently not correct.

**Comment by istiak101** [comment]
Screenshot from 2022-04-25 16-19-50
I am not sure which branch you're on. I tested this locally with 1.16.6. But in no case username would flow to next line as I am checking currently with different viewports and devices on browser dev tools.

**Comment by Gusted** [comment]
> I am not sure which branch you're on. I tested this locally with 1.16.6. But in no case username would flow to next line as I am checking currently with different viewports and devices on browser dev tools.

You're currently targeting the main branch(which is 1.17), could be just me, but the current layout doesn't seem to be correct.

**Comment by istiak101** [comment]
> You're currently targeting the main branch(which is 1.17), could be just me, but the current layout doesn't seem to be correct.

I would deploy 1.17 just in case if I can reproduce this. I will send css patch if I can with this same PR.

File: templates/user/profile.tmpl [hunk 1]
```diff
@@ -17,6 +17,9 @@
 						{{if .Owner.FullName}}<span class="header text center">{{.Owner.FullName}}</span>{{end}}
 						<span class="username text center">{{.Owner.Name}}</span>
 						<a href="{{.Owner.HomeLink}}.rss"><i class="ui grey icon tooltip ml-3" data-content="{{.i18n.Tr "rss_feed"}}" data-position="bottom center">{{svg "octicon-rss" 18}}</i></a>
+						<i class="ui grey icon tooltip ml-3">{{svg "octicon-person" 18}}</i>
+						<a href="{{.Owner.HomeLink}}?tab=followers">{{.Owner.NumFollowers}} {{.i18n.Tr "user.followers"}}</a> .
+						<a href="{{.Owner.HomeLink}}?tab=following">{{.Owner.NumFollowing}} {{.i18n.Tr "user.following"}}</a>
```
Comment (silverwind) [inline templates/user/profile.tmpl:22]:
```suggestion
						<a class="muted" href="{{.Owner.HomeLink}}?tab=followers">{{svg "octicon-person" 18 "mr-2"}}{{.Owner.NumFollowers}} {{.i18n.Tr "user.followers"}}</a> · <a class="muted" href="{{.Owner.HomeLink}}?tab=following">{{.Owner.NumFollowing}} {{.i18n.Tr "user.following"}}</a>
```

image

**Comment by istiak101** [comment]
I built the main branch. The CSS is indeed incorrect on 1.17(ba7750d6e). 1.16 branch is fine. I will check the CSS diff on both on both branch.

File: templates/user/profile.tmpl (hunk 1 above)
Comment (istiak101) [reply templates/user/profile.tmpl:22]: Thank you for your suggestion. I really needed the "muted" class.

**Review by silverwind — APPROVED** [review]

**Review by Gusted — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Comment by wxiaoguang** [comment]
Hmmm ....... it's broken ......

https://try.gitea.io/wxiaoguang

image

File: templates/user/profile.tmpl (hunk 1 above)
Comment (wxiaoguang) [reply templates/user/profile.tmpl]: Is there anything missing? The UI on try.gitea.io is not the same as the screenshot.

**Comment by silverwind** [comment]
Ooops, I forgot to suggest the wrapper div, fix in https://github.com/go-gitea/gitea/pull/19805.

**Pull request merged**